python-interface/src/MiniBotFramework/Communication/TCP.py

```diff
@@ -29,33 +35,49 @@ def get_command(self):
         self.command = ""
         return temp
 
-def run():
-    while TCP.tcp is None:
-        time.sleep(1)
-    while True:
-        print("Waiting for connection")
-        connectionSocket, addr = TCP.tcp.server_socket.accept()
-        connectionSocket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-        print("Connection accepted")
-        active=True
-        while active:
-            command = ""
-            while active:
-                try:
-                    lastLen = len(command)
-                    command += connectionSocket.recv(1024).decode()
-                    if lastLen == len(command):
+    def send_to_basestation(self, key, value):
+        """
+        Sends information back to the basestation. can only execute if the
+        connection is active
+        """
+        if self.active:
+            # connection is active, send
+            try:
+                message = "<<<<" + key + ":" + value + ">>>>"
```

Comment:
I have to separate PUT_IP SwarmMaster and 192.168.4.21. How do I do it? It's currently configured as PUT_IP:SwarmMaster,192.168.4.21 (maybe I can switch both separators)